data/experiment-recipe-samples/desktop-97-rollout.json

```diff
@@ -0,0 +1,48 @@
+{
+  "appId": "firefox-desktop",
+  "appName": "firefox_desktop",
+  "application": "firefox-desktop",
+  "arguments": {},
+  "isRollout": true,
+  "branches": [
+    {
+      "feature": {
+        "enabled": true,
+        "featureId": "urlbar",
+        "value": {
+          "firefoxSuggestLabelsEnabled": true,
+          "quickSuggestEnabled": false,
+          "quickSuggestNonSponsoredIndex": -1,
+          "quickSuggestShouldShowOnboardingDialog": true,
+          "quickSuggestShowOnboardingDialogAfterNRestarts": 2,
+          "quickSuggestSponsoredIndex": -1
+        }
+      },
+      "ratio": 1,
+      "slug": "control"
+    }
+  ],
+  "bucketConfig": {
+    "count": 1700,
+    "namespace": "urlbar-21-rollout",
+    "randomizationUnit": "normandy_id",
+    "start": 7300,
+    "total": 10000
+  },
+  "channel": "release",
+  "endDate": null,
+  "featureIds": ["urlbar"],
+  "id": "desktop-rollout-firefox-suggest-history-rollout-3",
+  "isEnrollmentPaused": true,
+  "outcomes": [],
+  "probeSets": [],
+  "proposedDuration": 28,
+  "proposedEnrollment": 7,
+  "referenceBranch": "control",
+  "schemaVersion": "1.7.0",
```

Comment:
No idea what the eventual schemaVersion should be, here. It depends on what PR lands first, and we'd like to remove it anyway per issue #186 - but we need *something* here to pass CI in the meantime